packages/resource-deployment/templates/batch-account.template.json

```diff
@@ -36,6 +36,34 @@
             "metadata": {
                 "description": "Name of the virtual network to be used for the batch pool nodes"
             }
+        },
+        "scanRequestPoolNodes": {
```

Comment:
we are depreciating legacy workflow and can drop `scanRequestPoolNodes` and `urlScanPoolNodes` completely

Reply:
I removed the parameters for the legacy pools and hardcoded them to 1, but since removing the legacy flow is going to change more files I think that should be a seperate PR